utils/__tests__/preview_helpers.test.js

```diff
@@ -0,0 +1,38 @@
+import { isPathValid } from '../preview';
+
+import * as path from 'path';
+
+describe('Test preview server helper functions', () => {
+    test('Test isPathValid returns true for valid paths', async () => {
```

Comment:
If `isPathValid` is no longer async, this can be removed
```suggestion
    test('Test isPathValid returns true for valid paths', () => {
```